jupyter_telemetry/eventlog.py

```diff
@@ -139,10 +139,14 @@ def record_event(self, schema_name, version, event):
         schema = self.schemas[(schema_name, version)]
         jsonschema.validate(event, schema)
 
+        # Incremented every time we change the capsule itself
+        telemetry_version = 1
```

Comment:
Perhaps this variable should be public facing? We can make it a global variable, i.e. `TELEMETRY_VERSION`, at the top of this module that's importable by other libraries. I'm thinking other libraries might need this variable for writing tests...?